Gordon360/ApiControllers/AccountsController.cs

```diff
@@ -448,30 +450,13 @@ public IHttpActionResult AdvancedPeopleSearch(bool includeAlumniSearchParam, str
             var viewerType = _roleCheckingService.getCollegeRole(viewerName);
 
             // Create accounts viewmodel to search
-            var accountsWithoutCurrentStudents = Data.AllPublicAccountsWithoutCurrentStudents;
-            var accountsWithoutAlumni = Data.AllPublicAccountsWithoutAlumni;
-            var accounts = Data.AllPublicAccounts;
+            IEnumerable<JObject> accounts = new List<JObject>();

```

Comment:
Good checking for student or alumni even on the backend! This cleans up a lot of redundancy!